### ISSUE TITLE
Custom dimension datapacks do not work with OpenLoader installed

### ISSUE THREAD

**Issue by RisingInIris2017** [Darkhax-Minecraft/Open-Loader#23]
### Minecraft Version
1.16.5
### Forge Version
36.2.31
### OpenLoader Version
9.0.3
### How to reproduce the problem
1. In an instance without OpenLoader, create a world, and add a custom dimension datapack into its `datapacks` folder. For my testing, I used [this datapack](https://www.curseforge.com/minecraft/customization/seasons-dimension-datapack/files/3096441).
2. Enter this world, then type command `/execute in` and press `Tab`. It is expected that you should see something like `seasons:xxx` in the auto-complete list, and you can teleport to this dimension. In this case the datapack is working properly.
3. Install OpenLoader.
4. After OpenLoader is installed, you can enter the world you created in the Step 1, and find that the custom dimension is still accessable, the datapack is still working properly in that world.
5. But **none of new worlds you create after the OpenLoader being installed** will let you teleport to custom dimension, even though you can still see your custom dimension datapack is loaded and available with `/datapack list` command. You may try two ways: one is adding the same datapack into new world's `datapacks` folder(the Vanilla way), the other is adding the datapack into `\openloader\data`(the OpenLoader way). Neither of them works.
6. Disable or remove OpenLoader.
7. Create a new world and add the custom dimension datapack. Enter this world, then type command `/execute in` and press `Tab`. Something like `seasons:xxx` is seen in the auto-complete list, and you can teleport to this dimension. Datapack works properly again.

In short, once you installed OpenLoader, though custom dimension datapacks can still be loaded and available, but custom dimensions they attempt to add are all unaccessable. Removing OpenLoader instantly kills this issue, but I do need OpenLoader for my modpack.

I cannot figure out the exactly reason of this issue, so I think post it here will be helpful.
Thanks in advance for your reply and kind-hearted help.

**Comment by Darkhax** [comment]
Hello,

I have tried to recreate this issue using the steps you described but everything seems to work fine. I am a little confused with step 5 though. When you create a new world are you doing anything to install the seasons dimension? When I run `/datapack list` it does not show the seasons pack. If I install seasons through the vanilla folder or OpenLoader it appears as expected.

Please check your log files for any errors related to OpenLoader or the seasons pack. It's possible you have done something to install the pack twice, or another datapack or mod you have installed is creating issues.

**Comment by RisingInIris2017** [comment]
> Hello,
> 
> I have tried to recreate this issue using the steps you described but everything seems to work fine. I am a little confused with step 5 though. When you create a new world are you doing anything to install the seasons dimension? When I run `/datapack list` it does not show the seasons pack. If I install seasons through the vanilla folder or OpenLoader it appears as expected.
> 
> Please check your log files for any errors related to OpenLoader or the seasons pack. It's possible you have done something to install the pack twice, or another datapack or mod you have installed is creating issues.

Thanks for your help! I have retried to load the pack with OpenLoader again after I deleted the `\openloader` folder. This time it works properly, and I confirmed that the issue is due to a certain mistake I had made.

So I will close the issue. Thank you again!

**Comment by Sunconure11** [comment]
@Darkhax Sorry to necro, but is this still present with 1.19.2, or is there an error in my pack? I can send over a copy for evaluation, if needed. It does use a few mods for some of the blocks, but that is not the primary issue going on, as far as I can tell.
